Assets/_BuildingBlocks/Credentials/Scripts/CredentialHandler.cs

```diff
@@ -0,0 +1,82 @@
+using Netherlands3D.Twin.Layers.LayerTypes.Credentials.Properties;
+using System;
+using Netherlands3D.Credentials.StoredAuthorization;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Netherlands3D.Credentials
+{
+    //this is the handler for the ui flow to check credentials
+    //will be active in the layerpanel when trying to add a layer through url
+    //we also use this credentialhandler for storing credential data while not having active layers yet
+    public class CredentialHandler : MonoBehaviour, ICredentialHandler
```

Comment:
kan deze nog gerenamed worden? zodat het duidelijk is dat het om de UI handler gaat en niet de LayerHandler?